.pre-commit-config.yaml

```diff
@@ -45,3 +45,13 @@ repos:
         language_version: python3
         additional_dependencies: ['types-setuptools', 'types-python-dateutil',
         'types-requests', 'types-chardet', 'types-six']
+  # Check-manifest: ensures required non-Python files are included in MANIFEST.in
+  # https://github.com/mgedmin/check-manifest/blob/master/.pre-commit-hooks.yaml
+  - repo: https://github.com/mgedmin/check-manifest
+    rev: "0.48"
+    hooks:
+      - id: check-manifest
+        additional_dependencies: ['h5py', 'wheel', 'future', 'numpy', 'pandas',
+        'python-dateutil', 'pytz', 'six', 'pyarrow', 'chardet', 'fastavro',
+        'python-snappy', 'charset-normalizer', 'psutil', 'scipy', 'requests',
+        'networkx','typing-extensions']
```

Comment:
have to add these for `check-manifest` to run in the `pre-commit` environment

Reply:
read [this](https://github.com/pre-commit/pre-commit/issues/730) but was reading a post from the creator of `pre-commit` that using `-r requirements.txt` in the `add_dependencies` is not good practice